web/themes/contrib/civictheme/civictheme.api.php

```diff
@@ -59,4 +60,37 @@ function hook_civictheme_automated_list_preprocess_view_alter(array &$variables,
     $variables['with_background'] = TRUE;
     $variables['vertical_spacing'] = 'both';
   }
+
+  /**
+   * Allow to suppress page regions for pages with Layout Builder enabled.
+   *
+   * @param array $variables
+   *   Array of variables passed to the page template.
+   * @param array $context
+   *   Array of context data.
+   *   - node: The node object.
+   *   - layout_builder_settings_per_view_mode: An array of the layout builder
+   *     settings keyed by view mode.
+   *
+   * @SuppressWarnings(PHPMD.StaticAccess)
+   */
+  function hook_civictheme_layout_suppress_page_regions_alter(array &$variables, array $context): void {
+    /** @var \Drupal\node\NodeInterface $node */
+    $node = $variables['node'];
+    if ($node->bundle() == 'civictheme_page' && $context['layout_builder_settings_per_view_mode']['full']['enabled']) {
```

Comment:
It actually goes deeper than this - each node can have their own layout - does this layout work on a node rendering level or theme registry setup level?

Reply:
it is up to a consumer theme to decide how to handle the logic - it is based on the caching mehcanism. the passed `$context['layout_builder_settings_per_view_mode']` is a shorthand for the node's config. a consumer site can retrieve more informatin on per-node basis (if required) from `$variables`